lib/inventory_refresh/save_collection/saver/base.rb

```diff
@@ -269,14 +269,8 @@ def assert_referential_integrity(hash)
         true
       end
 
-      # @return [Time] A rails friendly time getting config from ActiveRecord::Base.default_timezone (can be :local
-      #         or :utc)
       def time_now
-        if ActiveRecord::Base.default_timezone == :utc
-          Time.now.utc
-        else
-          Time.zone.now
-        end
+        Time.current
```

Comment:
@agrare this handles if Time.now is not set (nil) and returns Time.now which should be localtime if not set.  If set to utc, it will return utc time.

See: https://github.com/rails/rails/blob/v6.1.7.7/activesupport/lib/active_support/core_ext/time/calculations.rb#L39-L41